core/commands.c

```diff
@@ -328,9 +329,107 @@ static int killCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[])
 
 static int pushCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]) {
   return snprintf(outbuf, outbufc, "Push Command Not Allowed");
-  
+
 }
 
+static int flightProfileGetCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]){
+  //Returns flight profile information
+  pod_t *pod = get_pod();
+  flight_profile_t *profile = &pod->flight_profile;
+  return snprintf(outbuf, outbufc, "watchdog_timer: %i\n"
+                                   "emergency_hold: %i\n"
+                                   "braking_wait: %i\n"
+                                   "pusher_timeout: %i\n"
+                                   "pusher_state_accel_min: %f\n"
+                                   "pusher_state_min_timer: %i\n"
+                                   "pusher_distance_min: %f\n"
+                                   "primary_braking_accel_min: %f\n",
+                                    profile->watchdog_timer, profile->emergency_hold,
+                                    profile->braking_wait, profile->pusher_timeout,
+                                    profile->pusher_state_accel_min, profile->pusher_state_min_timer,
+                                    profile->pusher_distance_min, profile->primary_braking_accel_min);
+}
+
+static int flightProfileCommand(size_t argc, char *argv[], size_t outbufc, char outbuf[]){
+  //Allows user to configure flight profiles
+  pod_mode_t mode = get_pod_mode();
+  if(mode == 1){ //Only allowed in Boot mode
```

Comment:
This should be 

```
if (mode == Boot) {
```
Technically Boot Mode equates to `1` but the whole point of the `enum` is to disambiguate that